run_unittests.py

```diff
@@ -2957,6 +2960,14 @@ def test_reconfigure(self):
         self.build()
         self.run_tests()
 
+    def test_wipe_from_builddir(self):
+        testdir = os.path.join(self.common_test_dir, '162 custom target subdir depend files')
+        self.init(testdir)
+        self.__reconfigure()
+
+        os.chdir(self.builddir)
+        out = self.init(testdir, extra_args=['--wipe'])
```

Comment:
__[Flake8]__ <sub>posted by Sider</sub>

[F841] local variable 'out' is assigned to but never used